mysql-compatibility.md

```diff
@@ -113,6 +113,7 @@ TiDB 中，所有支持的 DDL 变更操作都是在线执行的。与 MySQL 相
 * 分区表支持 `HASH`、`RANGE` 和 `LIST` 分区类型。对于不支持的分区类型，TiDB 可能会报 `Warning: Unsupported partition type %s, treat as normal table` 错误，其中 `%s` 为不支持的具体分区类型。
```

Comment:
“，”，“,” please check if the comma is in half-size?

Reply:
According to the PingCAP writing style, we should use commas in full-size between Chinese characters and commas in half-size between English characters. 
https://github.com/pingcap/docs-cn/blob/master/resources/pingcap-style-guide-zh.pdf
![image](https://user-images.githubusercontent.com/79440533/201571875-2173ae1b-2d8e-4885-b48b-f635855b5a49.png)